bamboo/create-release.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
For a sample of what this script *does*, you can take a look at the release I generated in testing on my fork: 

https://github.com/Jkovarik/cumulus/releases